src/NetworkTestWidget/Tests/Tests.ts

```diff
@@ -1,12 +1,15 @@
 import { testBitrate, BitrateTest } from '@twilio/rtc-diagnostics';
 import { Device, Connection, PreflightTest } from 'twilio-client';
+import log from 'loglevel';
 import { regionalizeIceUrls } from '../../utils';
 import { Region } from '../../types';
 import RTCSample from 'twilio-client/es5/twilio/rtc/sample';
 
+log.setLevel(process.env.NODE_ENV === 'development' ? 'debug' : 'error', false);
```

Comment:
This uses a global logger for the entire application. We should name our logger so it doesn't clash with other libraries who might be using loglevel. Please see below, feel free to update or change it.
```ts
import { getLogger } from 'loglevel';
import { Device } from 'twilio-client`;

// Initialize the logger.
const log = getLogger(my_react_app_package_name);
const level = process.env.NODE_ENV === 'development' ? 'debug' : 'error';

// Set the log level for the app
log.setLevel(level);

// Set the log level for client js
getLogger(Device.packageName).setLevel(level);
```